tests/lib/remote_test.sh

```diff
@@ -264,6 +265,29 @@ function test_ssh_connection_failure_message()
   assertEquals "($LINENO):" "$no_config_file_failure_message" "$ret"
 }
 
+test_ssh_connection_failure_message_with_bad_formatted_remote_config()
```

Comment:
For consistency, use the function keyword to define a function

Reply:
Thanks a lot for the catch, @magalilemes!
Now that you mentioned it, it would be nice to have a ShellCheck rule for this one. However, I only found the opposite 🤣 https://www.shellcheck.net/wiki/SC2112.